aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -846,7 +846,9 @@ AT_ERROR("lu: MAGMA library not found in "
     "compilation. Please rebuild with MAGMA.");
 #else
   auto self_data = self.data_ptr<scalar_t>();
+  magma_int_t m = magma_int_cast(self.size(-2), "m");
   magma_int_t n = magma_int_cast(self.size(-1), "n");
+  magma_int_t k = magma_int_cast(std::min(self.size(-2), self.size(-1)), "k");
```

Comment:
Couldn't we just do `std::min(m, n)` ?

Reply:
When considering https://github.com/maxhutch/magma/blob/master/include/magma_types.h#L39-L44, then yes.
Fixed.